tests/helpers.js

```diff
@@ -64,50 +68,79 @@ class FuzzTest {
 	}
 
 	execute() {
-		if (this.jestTestFile !== "") {
+		if (this.jestTestFile) {
 			this.executeWithJest();
-			return;
+		} else {
+			this.executeWithCli();
 		}
+	}
+
+	executeWithCli() {
 		const options = ["jazzer", this.fuzzFile];
 		options.push("-f " + this.fuzzEntryPoint);
 		if (this.sync) options.push("--sync");
+		if (this.coverage) options.push("--coverage");
+		if (this.dryRun !== undefined) options.push("--dryRun=" + this.dryRun);
+		for (const include of this.includes) {
+			options.push("-i=" + include);
+		}
+		for (const exclude of this.excludes) {
+			options.push("-e=" + exclude);
+		}
 		for (const bugDetector of this.disableBugDetectors) {
 			options.push("--disable_bug_detectors=" + bugDetector);
 		}
-
-		if (this.customHooks) {
-			options.push("--custom_hooks=" + this.customHooks);
+		for (const customHook of this.customHooks) {
+			options.push("--custom_hooks=" + customHook);
 		}
-		options.push("--dryRun=" + this.dryRun);
-		if (this.coverage) options.push("--coverage");
 		options.push("--");
-		options.push("-runs=" + this.runs);
+		if (this.runs !== undefined) options.push("-runs=" + this.runs);
 		if (this.forkMode) options.push("-fork=" + this.forkMode);
-		options.push("-seed=" + this.seed);
+		if (this.seed) options.push("-seed=" + this.seed);
 		for (const dictionary of this.dictionaries) {
 			options.push("-dict=" + dictionary);
 		}
 		this.runTest("npx", options, { ...process.env });
 	}
 
 	executeWithJest() {
-		// Put together the libfuzzer options.
-		const fuzzerOptions = ["-runs=" + this.runs, "-seed=" + this.seed];
+		const fuzzerOptions = [];
+		if (this.runs) {
+			fuzzerOptions.push("-runs=" + this.runs);
+		}
+		if (this.seed) {
+			fuzzerOptions.push("-seed=" + this.seed);
+		}
 		const dictionaries = this.dictionaries.map(
 			(dictionary) => "-dict=" + dictionary,
 		);
 		fuzzerOptions.push(...dictionaries);
 
-		// Put together the jest config.
-		const config = {
-			sync: this.sync,
-			include: [this.jestTestFile],
-			disableBugDetectors: this.disableBugDetectors,
-			fuzzerOptions: fuzzerOptions,
-			customHooks: this.customHooks,
-			dryRun: this.dryRun,
-			mode: this.jestRunInFuzzingMode ? "fuzzing" : "regression",
-		};
+		const config = {};
```

Comment:
Could this all be replaced by one check if the key's value is undefined when writing out the config?

Reply:
I don't think I can feel comfortable with you having positive code line count contributions  :laughing:

Reply:
As it's serialized by `JSON.stringify` this seemed the most obvious way. How would that look like?